src/backend/routers/case_manager.ts

```diff
@@ -164,7 +164,7 @@ export const case_manager = router({
       const para = await createPara(
         req.input,
         req.ctx.db,
-        req.ctx.auth.userId,
+        req.ctx.auth.session.user?.name ?? "",
         req.ctx.env.EMAIL,
         req.input.email,
         req.ctx.env
```

Comment:
This is a lot of arguments, whose values don't look like the names of the parameters. Can we consider changing this function to take keyword arguments, so that it's clear for example that `req.ctx.auth.session.user?.name ?? ""` is being fed as case_manager_name?

Reply:
ah via destructuring? is that increasingly common nowadays?

Reply:
thanks, I'll open up another PR for this.